### PR TITLE
[ci_updates] pylint + Python3 compatibility

### PR DESCRIPTION
**Pull request by LoyVanBeek**
- [x] `cob_gazebo_tools`

### CODE REVIEW

File: cob_gazebo_tools/scripts/remove_object.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def get_flat_dict(objects, parent_name):
         try:
             rospy.wait_for_service('/gazebo/delete_model')
             res = srv_delete_model(name)
-        except rospy.ServiceException, e:
+        except rospy.ServiceException as e:
```
Comment (LoyVanBeek) [inline cob_gazebo_tools/scripts/remove_object.py]: Probably pylint will whine about the unused `e` variable

File: cob_gazebo_tools/scripts/spawn_object.py [hunk 2]
```diff
@@ -215,7 +215,7 @@ def get_flat_dict(objects, parent_name):
             srv_delete_model = rospy.ServiceProxy('gazebo/delete_model', DeleteModel) # TODO this service causes gazebo (current groovy version) to crash
             try:
                 res = srv_delete_model(key)
-            except rospy.ServiceException, e:
+            except rospy.ServiceException as e:
```
Comment (LoyVanBeek) [inline cob_gazebo_tools/scripts/spawn_object.py]: Pylint will complain about the unused `e`.

File: cob_gazebo_worlds/scripts/tf_publisher.py [hunk 3]
```diff
@@ -31,7 +31,7 @@
         
         try:
             rate.sleep()
-        except rospy.ROSTimeMovedBackwardsException, e:
+        except rospy.ROSTimeMovedBackwardsException as e:
```
Comment (LoyVanBeek) [inline cob_gazebo_worlds/scripts/tf_publisher.py]: Pylint will complain about the unused `e`.

**Comment by fmessmer** [comment]
- `kinetic` job:
   - `pylint`: :heavy_check_mark: 
   - `pylint3`: `E0602(undefined-variable)`
 - `melodic` job:
   - `pylint`: :heavy_check_mark: 
   - `pylint3`: `E0401(import-error)`, `E0602(undefined-variable)`

**Review by LoyVanBeek — COMMENTED** [review]
LGTM

**Review by fmessmer — APPROVED** [review]

**Pull request merged**